doc/contributing/documentation_strings.rst

```diff
@@ -24,6 +24,26 @@ Example::
     /// @param delimiter        A delimiter we're using to split the text.
     /// @param max_items        Limit number of splits to produce a vector containing up to ``max_items`` items.
```

Comment:
You have double backticks here around `max_items`. Seems to me single backticks work the same?

I think we should make it a separate point to highlight people can and should use backticks to mark identifiers etc., it increases readability quite a bit.

Reply:
Double backticks fixed, a note about using backticks added.
Please note I had to use \`\`\`backticks\`\`\` in the rst to get it rendered as expected: \`backticks\`